web/src/components/navbar/NavigatorLogo.tsx

```diff
@@ -10,14 +9,16 @@ interface Props {
 export const NavigatorLogo = (props: Props): ReactElement => {
   const { Config } = useConfig();
   return (
-    <Link href={ROUTES.dashboard} passHref>
-      <a href={ROUTES.dashboard}>
-        <img
-          className={props.useSmall ? "height-3" : "height-4"}
-          src="/img/Navigator-logo@2x.png"
-          alt={Config.pagesMetadata.titlePrefix}
-        />
-      </a>
+    <Link href={Config.navigationDefaults.navBarBusinessNJGovHomePageLink} passHref>
+      <div className={"height-4"}>
+        <a href={Config.navigationDefaults.navBarBusinessNJGovHomePageLink}>
+          <img
+            className={props.useSmall ? "height-3" : "height-4"}
+            src="/img/business.NJ.gov-logo.svg"
+            aria-label={"Business.nj.gov Home"}
```

Comment:
My understanding is that `aria-label` is used for interactive elements. Since we're making the logo clickable to route to the dashboard, we should have it on the image.

Reply:
I can make it part of the CMS